index.bs

```diff
@@ -560,76 +586,535 @@ message? -->
+
+
+If the controller wishes to modify the state of the remote playback (for
+example, to pause, resume, skip, etc), it may send a
+remote-playback-modify-request message with the following values:
+
+: remote-playback-id 
+:: The ID of the remote playback to be modified.
+
+: controls
+:: Updated controls as defined in {#remote-playback-state-and-controls}
+
+When a receiver receives a remote-playback-modify-request it should send a
+remote-playback-modify-response message in reply with the following values:
+
+: state
+:: The updated state of the remote playback as defined in
+    [[#remote-playback-state-and-controls]].
+
+When the state of remote playback changes without request for modification from
+the controller (such as when the skips or pauses due to user user interaction on
+the receiver), the receiver may send a remote-playback-state-event to the
+controller.
+
+: remote-playback-id 
+:: The ID of the remote playback whose state has changed.
+
+: state
+:: The updated state of the remote playback, as defined in
+    [[#remote-playback-state-and-controls]].
+
+
+To terminate the remote playback, the controller may send a
+remote-playback-termination-request message with the following values:
+
+: remote-playback-id 
+:: The ID of the remote playback to terminate.
+
+: reason
+:: The reason the remote playback is being terminated.
+
+When a receiver receives a presentation-termination-request, it should
+send back a presentation-termination-response message to the controller.
+
+If a receiver terminates a remote playback without a request from the controller
+to do so, it must send a remote-playback-termination-event message to the
+controller with the following values:
+
+: remote-playback-id
+:: The ID of the remote playback that was terminated.
+
+: reason
+:: The reason the remote playback was terminated.
+
+As mentioned in
+[[REMOTE-PLAYBACK#disconnecting-from-a-remote-playback-device|Remote Playback
+API section 6.2.7]], terminating the remote playback means the controller is no
+longer controlling the remote playback and does not necessarily stop media from
+rendering on the receiver.  Whether or not the receiver stops rendering media depends
+upon the implmentation of the receiver.
+
+
+
+Remote Playback State and Controls {#remote-playback-state-and-controls}
+------------------------------------------------------------------------
+
+In order for the controller and receiver to stay in sync with regards to the
+state of the remote playback, the controller may send controls to modify the state
+(for example, via the remote-playback-modify-request message) and the receiver
+may send updates about state changes (for example, via the
+remote-playback-state-event message).
+
+The controls sent by the controller include the follwing individual control
+values, each of which is optional.  This allows the controller to change one
+control value or many control values at once without having to specify all
```

Comment:
The media element may be change in several different ways within the same JS execution in way that I think we should consider like a transaction (for example, seek and resume, or resume and change playback rate).  I believe we discussed this at TPAC.

It's a good question about order of application.  Can you think of a scenario where the order of application matters?

If the controller really wants to, it can always send separate messages, of course.